main.js

```diff
@@ -0,0 +1,110 @@
+// Select elements
+const inputSearch = document.querySelector('#input-search');
+const btnSearch = document.querySelector('#btn-search');
+const resultBox = document.querySelector('.results-box');
+const resultInnerBox = document.querySelector('.results-inner-box');
+const countryFlag = document.querySelector('.country-flag');
+const countryHeader = document.querySelector('.country-header');
+const countryInfo = document.querySelector('.country-info');
+const errorMessage = document.querySelector('.error-message');
+const spinner = document.querySelector('#spinner');
+
+// API call
+const getCountry = async function (cntry) {
+  resultBox.classList.add('hidden');
+  spinner.removeAttribute('class');
+
+  try {
+    const response = await axios.get(
+      `https://restcountries.eu/rest/v2/name/${cntry}?fullText=true`
+    );
+    createSearchResult(response.data[0]);
+  } catch (error) {
+    showError();
+  }
+
+  spinner.setAttribute('class', 'hideSpinner');
+};
+
+// returns all currencies as string
+const currencies = currency =>
+  currency
+    .map((curr, i, arr) => {
+      return i + 1 < arr.length ? `${curr.name}'s and` : `${curr.name}'s`;
+    })
+    .join(' ');
+
+// Returns string with spoken languages
+const languages = language =>
```

Comment:
Functienaam iets aanpassen waardoor de property naam ook in meervoudsvorm kan (want het is een array en niet één language):
`const getLanguagesString = languages =>`